### PR TITLE
Fix/history bug

### LINKED ISSUE #707: Unexpected behavior with back button?
Expected behavior:

Clicking the back button in the widget causes the widget itself to go back to the previous state. I.e. I click back from this page:

image

and am taken back to this page:

image

Actual behavior:

Clicking the back button from any place in the widget goes back to the page before the widget.

-----

Is this actually the intended behavior? I want to make sure before I put too much time into trying to debug something which might be intentional.

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Currently, if both the SDK and the host app use manipulate the DOM history, the SDK back button might wrongly interact with the host app history.

# Solution
Use `createMemoryHistory` instead of `createBrowserHistory`. This changes fixes #707 

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ _put `n/a` if item is not relevant to PR changes_
 
 - [ ] Has the CHANGELOG been updated?
 - [ ] Has the README been updated?
+- [ ] Has the CONTRIBUTING been updated?
```
Comment (kopijunkie) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: Minor: This sounds odd. Maybe use `Has the CONTRIBUTING doc been updated` instead.

Comment (stefaniacardenas) [reply .github/PULL_REQUEST_TEMPLATE.md]: True. Fixed

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1004-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1004-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**